docs/concepts/workloads/controllers/jobs-run-to-completion.md

```diff
@@ -219,17 +219,17 @@ When a Job completes, no more Pods are created, but the Pods are not deleted eit
 they don't show up with `kubectl get pods`, but they will show up with `kubectl get pods -a`.  Keeping them around
 allows you to still view the logs of completed pods to check for errors, warnings, or other diagnostic output.
 The job object also remains after it is completed so that you can view its status.  It is up to the user to delete
-old jobs after noting their status.  Delete the job with `kubectl` (e.g. `kubectl delete jobs/pi` or `kubectl delete -f ./job.yaml`).  When you delete the job using `kubectl`, all the pods it created are deleted too.
+old jobs after noting their status.  Delete the job with `kubectl` (e.g. `kubectl delete jobs/pi` or `kubectl delete -f ./job.yaml`). When you delete the job using `kubectl`, all the pods it created are deleted too.
 
-If a Job's pods are failing repeatedly, the Job will keep creating new pods forever, by default.
-Retrying forever can be a useful pattern.  If an external dependency of the Job's
-pods is missing (for example an input file on a networked storage volume is not present), then the
-Job will keep trying Pods, and when you later resolve the external dependency (for example, creating
-the missing file) the Job will then complete without any further action.
+By default, a Job will run uninterrupted unless a Pod fails, at which point the Job defers to the
+`.spec.backoffLimit` described above. Another way to terminate a Job is by setting an active deadline.
+Do this by setting the `.spec.activeDeadlineSeconds` field of the Job to a number of seconds.
 
-However, if you prefer not to retry forever, you can set a deadline on the job.  Do this by setting the
-`spec.activeDeadlineSeconds` field of the job to a number of seconds.  The job will have status with
-`reason: DeadlineExceeded`.  No more pods will be created, and existing pods will be deleted.
+When `activeDeadlineSeconds` is set, the Job will terminate the active Pod once the deadline is reached.
+If the Job is in a failing (backoff) state, no additional Pods will be created even if the Job has not yet
+reached its `backoffLimit`. In both cases, once the `activeDeadlineSeconds` is reached, the Job will have a status with `reason: DeadlineExceeded`.
```

Comment:
Once we reach the `activeDeadlineSeconds` we terminate the job and all its pods. The result is that the job gets a condition with reason `DeadlineExceeded`.

Reply:
It's worth mentioning that ADS applies to overall duration of the job, no matter how many pods are created.